### PR TITLE
Include LICENSE in sdist and wheel

### LINKED ISSUE #6: Include LICENSE in sdist
cylc-xtriggers will likely become a conda package. Similarly as it was done in metomi-isodatetime, and is being done in cylc-flow, we will need the LICENSE file to be included in the sdist, so we don't need to duplicate the LICENSE file in both this repo and conda-recipe.

See https://github.com/metomi/isodatetime/pull/144 and https://github.com/cylc/cylc-flow/issues/3387 for more.

### PR DESCRIPTION
**Pull request by sgaist**
Fixes #6

### CODE REVIEW

**Comment by kinow** [comment]
Looking good! I'm just going to try `python setup.py sdist`. @sgaist could you take a look at the `CONTRIBUTING.md` file in the meantime, and if you are OK with that, add your name to the list of contributors, please?

Thanks!
Bruno

File: setup.cfg [hunk 1]
```diff
@@ -38,3 +38,6 @@ setup_requires = pytest-runner==4.4
 
 [aliases]
 test = pytest
+
+[metadata]
```
Comment (kinow) [inline setup.cfg]: Running `python setup.py sdist` failed with the following error:

```bash
$ python setup.py sdist
Traceback (most recent call last):
  File "setup.py", line 62, in <module>
    "Tracker": "https://github.com/cylc/cylc-xtriggers/issues"
  File "/home/kinow/Development/python/workspace/cylc-xtriggers/venv/lib/python3.7/site-packages/setuptools/__init__.py", line 144, in setup
    _install_setup_requires(attrs)
  File "/home/kinow/Development/python/workspace/cylc-xtriggers/venv/lib/python3.7/site-packages/setuptools/__init__.py", line 137, in _install_setup_requires
    dist.parse_config_files(ignore_option_errors=True)
  File "/home/kinow/Development/python/workspace/cylc-xtriggers/venv/lib/python3.7/site-packages/setuptools/dist.py", line 697, in parse_config_files
    self._parse_config_files(filenames=filenames)
  File "/home/kinow/Development/python/workspace/cylc-xtriggers/venv/lib/python3.7/site-packages/setuptools/dist.py", line 594, in _parse_config_files
    (parser.read_file if six.PY3 else parser.readfp)(reader)
  File "/home/kinow/anaconda3/lib/python3.7/configparser.py", line 717, in read_file
    self._read(f, source)
  File "/home/kinow/anaconda3/lib/python3.7/configparser.py", line 1065, in _read
    lineno)
configparser.DuplicateSectionError: While reading from 'setup.cfg' [line 42]: section 'metadata' already exists
```

Looks like the `[metadata]` is already declared above this line. Could you please move the `license_files` there @sgaist ? Then I'll do another quick test here :+1: 

Thanks!

**Review by kinow — CHANGES_REQUESTED** [review]

File: setup.cfg (hunk 1 above)
Comment (sgaist) [reply setup.cfg]: Late night coding...

I didn't pay enough attention to the file content.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ be able to consider large changes that appear out of the blue. New
 contributors should add their details to the [Code Contributors
 ](#code-contributors) section of this file as part of their first Pull
 Request, and reviewers are responsible for checking this before merging the
-new branch into *master*.
```
Comment (kinow) [inline CONTRIBUTING.md:26]: Harmless changes removing these trailing spaces. So all good IMO.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -35,6 +35,7 @@ below.
  - Matt Shin
  - Sadie Bartholomew
  - Bruno Kinoshita
+ - Samuel Gaist
```
Comment (kinow) [inline CONTRIBUTING.md:38]: Thanks for filling the CLA!

**Review by kinow — APPROVED** [review]
Generated `python setup.py sdist bdist_wheel` and the `LICENSE` file is now in both files. :+1: 

Thanks @sgaist !

File: setup.cfg (hunk 1 above)
Comment (kinow) [reply setup.cfg]: No worries, I code better in the morning, but also write some code in the late night... that I end up fixing in the morning :grin: let's merge it! Thanks!

**Pull request merged**